Makefile

```diff
@@ -317,6 +317,63 @@ clean-volumes:
 compose:
 	$(foreach MODULE, $(CANDIG_MODULES), $(MAKE) compose-$(MODULE);)
 
+#-- Temp --
+compose-authz:
+	docker-compose -f $(DIR)/lib/authz/docker-compose.yml up -d 2>&1
+
+compose-authz-down:
+	# closes primary authn and authz components
+	docker-compose -f $(DIR)/lib/authz/docker-compose.yml down
+	# closes the candig server along with its corresponding arbiter and opa 
+	docker-compose -f $(DIR)/lib/candig_server/docker-compose.yml down
+
+compose-authz-clean: compose-authz-down \
+	# needs sudo to run;
+	# for it to work, make sure your user does not need a password to use sudo
+
+	# clean keycloak
+	sudo rm -r $(DIR)/lib/authz/keycloak/data/* & 2>&1
```

Comment:
Keep in mind that the end-user of this deployment may not have access to elevated permissions (i.e. sudo). Furthermore, sudo should not be needed to remove local file directories...